README.md

```diff
@@ -160,7 +187,7 @@ cdef class MatrixArray:
         self.length = len(py_matrices)
         self.matrices = <SparseMatrix**>self.mem.alloc(self.length, sizeof(SparseMatrix*))
         for i, py_matrix in enumerate(py_matrices):
-            self.matrices[i] = sparse_matrix_init(self.mem, py_matrix)
+            self.matrices[i] = sparse_matrix_init_cymem(self.mem, py_matrix)
```

Comment:
It's minor, but this would have been better left as a separate PR, since it's unrelated to the topic of this PR, trivial to review/merge, and after that CI will run on this PR without approval.